vm-setup/roles/libvirt/tasks/vm_setup_tasks.yml

```diff
@@ -112,6 +112,17 @@
       when: item is failed
       with_items: "{{ vm_extradisks_check.results }}"
 
+    - name: Get vm uuid
```

Comment:
maybe a comment to say why we need the uuid for redfish ? (mapped to system-id)

Reply:
Done